src/views/SystemIntake/ContractDetails/index.tsx

```diff
@@ -165,57 +165,76 @@ const ContractDetails = ({
                       value
                     />
                     {values.fundingSource.isFunded && (
-                      <div className="width-card-lg margin-top-neg-2 margin-left-3 margin-bottom-1">
-                        <FieldGroup
-                          scrollElement="fundingSource.source"
-                          error={!!flatErrors['fundingSource.source']}
-                        >
-                          <Label htmlFor="IntakeForm-FundingSource">
-                            Funding Source
-                          </Label>
-                          <FieldErrorMsg>
-                            {flatErrors['fundingSource.source']}
-                          </FieldErrorMsg>
-                          <Field
-                            as={DropdownField}
+                      <div className="margin-top-neg-2 margin-left-3 margin-bottom-1">
```

Comment:
This whole section looks like a big change but all I did was add an extra div so that the link/help text added under the funding number could be outside of the "width-card-lg" attribute. This attibrute was causing the link text to be on three lines instead of one which I thought looked bad.